### ISSUE TITLE
Is there a way to download now-cli without bundled Node.js?

### ISSUE THREAD

**Issue by Siilwyn** [vercel/vercel#1602]
I'm using `now-cli` as a dependency but it takes up a whopping 100MB, I tried installing the repository but that doesn't give me any code to work with, just the download script. :(

**Comment by Siilwyn** [comment]
For posterity: so close yet so far, I'm stuck, using the GitHub tarball link in combination with `--ignore-scripts` it's possible to get the source. Sadly this package uses the `import / export` syntax which the latest Node.js version doesn't support yet.

```json
{
  "name": "nowrepo",
  "version": "1.0.0",
  "nowurl": "https://github.com/zeit/now-cli/tarball/11.4.6",
  "scripts": {
    "postinstall": "npm install --ignore-scripts --no-save $npm_package_nowurl"
  },
  "peerDependencies": {
    "now": "$npm_package_nowurl"
  }
}
```

I could transpile the code but that means I would have add another dependency plus it adds extra 'hacky' complexity. :/
Even when using webpack it doesn't work out of the box since the webpack config includes an `exclude` for `/node_modules/` and since this is done within `node_modules` it doesn't transpile any files. Now I can create a workaround for this but that makes it even more complex.

**Comment by Siilwyn** [comment]
@OlliV @leo
TL;DR: would you accept a PR to include the output of the build script for now-cli so it can be used without the bundled Node.js?

So I'm using the now-cli as a dependency which I know is not ideal but saves a lot of code e.g. the logic of which files to upload for a deployment. It works with the downside being the giant size since you probably know it is being packed with Node.js itself. So I'm trying to just get the JS, in particular the output in dist. After struggling with this I realised the most straightforward way would be to include the build output in the npm package and getting it with npm install --ignore-scripts now-cli. Thoughts?

**Comment by Siilwyn** [comment]
Because this whole project is set up to be bundled with `pkg` the `node_modules` directory is huge when just using `webpack` because all the dependencies are in `devDependencies` there is no way to differentiate.
